pkgs/by-name/le/leftwm-config/package.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  pname = "leftwm-config";
+  version = "0-unstable-2024-03-23";
+
+  src = fetchFromGitHub {
+    owner = "leftwm";
+    repo = "leftwm-config";
+    rev = "a9f2f21ece3a01d6c36610295ae3163644d3f99e";
+    hash = "sha256-wyb/26EyNyBJeTDUvnMxlMiQjaCGBES8t4VteNY1I/A=";
+  };
+
+  cargoHash = "sha256-fiB8y4v1gc8d2gvqsgdQ4xHIjzrLxcqp8ybk09tOO8U";
+
+  meta = {
+    broken = (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
I think the parenthesis are uneeded

Reply:
Oh yeah, is a remainder from originally. I think it kinda works for readability's sake, but I can remove it if preferred 😌 @Sigmanificient

Reply:
Prefer to wait for someone with more experience to give yu a better answer

Reply:
Resolving conversation due to below!